allennlp/service/predictors/semantic_role_labeler.py

```diff
@@ -42,11 +42,11 @@ def make_srl_string(words: List[str], tags: List[str]) -> str:
     def predict_json(self, inputs: JsonDict) -> JsonDict:
         sentence = inputs["sentence"]
         tokens = self.nlp.tokenizer(sentence)
-        text = TextField(tokens, token_indexers=self.token_indexers)
 
         results = {"verbs": []}  # type: JsonDict
         spacy_doc = self.nlp(sentence)
         words = [token.text for token in spacy_doc]
+        text = TextField(words, token_indexers=self.token_indexers)
```

Comment:
@schmmd with this fix the SRL model produces much more coherent outputs and matches what it is when you load it from a python script 👍